src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/FundingBandExcessModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+{
+    public class FundingBandExcessModel
+    {
+        private readonly int?[] _fundingBandCapsExceeded;
+
+        public FundingBandExcessModel(int apprenticesExceedingBand, IEnumerable<int?> fundingBandCapsExceeded)
+        {
+            _fundingBandCapsExceeded = fundingBandCapsExceeded.ToArray();
+            ApprenticesExceedingBand = apprenticesExceedingBand;
+        }
+
+        public int ApprenticesExceedingBand { get; }
```

Comment:
It wasn't immediately obvious to me that this was a count - I did figure this out by looking at the constructor calls but wonder if a better name could be found?